src/lib.rs

```diff
@@ -183,7 +182,7 @@ impl Mnemonic {
 	/// Performing this on a [Cow] means that all allocations for normalization
 	/// can be avoided for languages without special UTF8 characters.
 	#[inline]
-	#[cfg(feature = "std")]
+	#[cfg(any(feature = "alloc", feature = "std"))]
```

Comment:
Perhaps using `feature = "unicode-normalization"` would better show the reason for the feature gate?

Reply:
Yes, makes sense, I'll add it.